wagtail/embeds/finders/facebook.py

```diff
@@ -0,0 +1,118 @@
+import json
+import re
+
+from urllib import request as urllib_request
+from urllib.error import HTTPError, URLError
+from urllib.parse import urlencode
+from urllib.request import Request
+
+from django.conf import settings
+
+from wagtail.embeds.exceptions import EmbedException, EmbedNotFoundException
+
+from .base import EmbedFinder
+
+
+class AccessDeniedFacebookException(EmbedException):
```

Comment:
Just a little suggestion, which I am not quite sure about myself…

```suggestion
class AccessDeniedFacebookOEmbedException(EmbedException):
```

Would make this more explicit